.github/workflows/file-storage-create-release.yaml

```diff
@@ -20,7 +20,7 @@ jobs:
         uses: actions/setup-node@v1
         with:
           node-version: ${{ matrix.node-version }}
-      - run: npx lerna bootstrap --hoist
+      - run: npm install
```

Comment:
I think these should stay as bootstrap - if you make changes to both packages they won't be published yet so the tests will fail if it just pulls the version of deps from npm.

Reply:
Ah true, I'll copy across the things with the nohoist flag then

Reply:
If you check this one though (expand npm install step) it does do the bootstrap using the postinstall hook, with the benefit of copying the exclusions to all packages: https://github.com/Meeco/js-sdk/runs/1490190401

Reply:
Ah yes, sorry been a while since I've looked at the root `package.json` - forgot about that postinstall

https://github.com/Meeco/js-sdk/blob/master/package.json#L9